### ISSUE TITLE
Accessibility errors found in your template

### ISSUE THREAD

**Issue by wapyce** [gunhoon/gunhoon.github.io#1]
The user @carlsonsantana validate your site template "https://gunhoon.github.io" and found these accessibility errors:
* Page https://gunhoon.github.io/:
  * Issue *45236522-b63c-4146-a5a3-39349760f140*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.78:1. Recommendation: change text colour to #757575.
    * **Selector**: `html > body > footer > div > div > div:nth-child(3) > p`
    * **Context**: `<p>삽질하면서 배운 생각들을 정리하는 공간</p>`
  * Issue *29e570a2-1eba-418c-bc5f-72ffdb3f8255*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 4.15:1. Recommendation: change text colour to #2373db.
    * **Selector**: `html > body > footer > div > div > div:nth-child(2) > ul > li:nth-child(2) > a > span`
    * **Context**: `<span class="username">gunhoon</span>`
  * Issue *76c3dd14-04c7-47ab-8300-69b8ff77a5d8*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 4.15:1. Recommendation: change text colour to #2373db.
    * **Selector**: `html > body > footer > div > div > div:nth-child(2) > ul > li:nth-child(1) > a > span`
    * **Context**: `<span class="username">gunhoon</span>`
  * Issue *a200ae29-b767-4fc6-85d1-d2921a54617f*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 4.15:1. Recommendation: change text colour to #2373db.
    * **Selector**: `html > body > footer > div > div > div:nth-child(1) > ul > li:nth-child(2) > a`
    * **Context**: `<a class="u-email" href="mailto:gunhoon@gmail.com">gunhoon@gmail.com</a>`
  * Issue *addd47ba-49c7-4097-b731-00a9a34a8655*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.78:1. Recommendation: change text colour to #757575.
    * **Selector**: `html > body > footer > div > div > div:nth-child(1) > ul > li:nth-child(1)`
    * **Context**: `<li class="p-name">Gunhoon's Blog</li>`
  * Issue *17c74971-42d6-4a29-a06c-3fe4a4c7c37c*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 4.15:1. Recommendation: change text colour to #2373db.
    * **Selector**: `html > body > main > div > div > p > a`
    * **Context**: `<a href="/feed.xml">via RSS</a>`
  * Issue *efb6614b-6ffa-4d3c-b0a1-068ecb81aea1*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.78:1. Recommendation: change text colour to #757575.
    * **Selector**: `html > body > main > div > div > ul > li:nth-child(2) > span`
    * **Context**: `<span class="post-meta">Jul 24, 2018</span>`
  * Issue *0f5e01f1-30a8-4d00-9960-24c5e23c875f*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.78:1. Recommendation: change text colour to #757575.
    * **Selector**: `html > body > main > div > div > ul > li:nth-child(1) > span`
    * **Context**: `<span class="post-meta">Jul 25, 2018</span>`

You can check these accessibility errors using [pa11y](https://github.com/pa11y/pa11y).
You can view the [full validation results in our website](https://wapyce.herokuapp.com/validation/validations/b3535b3a-f002-485c-847b-b6de3e38bd2a/).
